bin/prettier.js

```diff
@@ -202,7 +204,7 @@ function format(input, opt) {
     return;
   }
 
-  return prettier.format(input, opt);
+  return prettier.formatWithCursor(input, opt);
```

Comment:
I'm really upset that we have to introduce a new api and didn't think about this when we built .format :(